content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Overview: Using Jupyter Notebooks in Kubeflow"
+description = "Why use Jupyter Notebooks in Kubeflow"
+weight = 5
++++
+
+
+The benefit of integrating Jupyter Notebooks service in Kubeflow is that:
+
+* In an enterprise environment, it integrates well with the rest of the infrastructure with respect to authentication and access control, and
+* It allows users to create notebook containers / pods in the cluster rather than locally on their laptops.
```

Comment:
Suggested change:

"It enables easier notebook sharing across the organization. Users can create notebook containers or pods directly in the cluster, rather than locally on their workstations. Admins can provide standard notebook images for their organization, and set up role-based access control (RBAC), Secrets and Credentials to manage which teams and individuals can access the notebooks."


I removed the detail about